src/backend/distributed/citus--7.2-1--7.2-2.sql

```diff
@@ -0,0 +1,24 @@
+/* citus--7.2-1--7.2-2 */
+
+CREATE TYPE citus.copy_format AS ENUM ('csv', 'binary', 'text');
+
+CREATE OR REPLACE FUNCTION pg_catalog.read_intermediate_result(key text, format citus.copy_format default 'csv')
+    RETURNS record
+    LANGUAGE C STRICT VOLATILE PARALLEL SAFE
```

Comment:
Just out of curiosity: I don't think `PARALLEL SAFE` would help us given that we're iterating rows one-by-one? Or, am I missing anything?

Reply:
Yes, but it allows postgres to call the function from within parallel plans, if necessary.